index.js

```diff
@@ -746,6 +748,50 @@ export default async function makeHyperFetch ({
     return serveFile(drive, path, isRanged)
   }
 
+  async function diffVersions (request) {
+    const url = new URL(request.url)
+    const { hostname, pathname: rawPathname } = url
+    const pathname = decodeURI(ensureLeadingSlash(rawPathname))
+
+    const parts = pathname.split('/')
+    const versions = parts[3]
+    let [oldVersion, newVersion] = versions.split(/\.\./).map(Number)
```

Comment:
```suggestion
    let [oldVersion, newVersion] = versions.split('...').map(Number)
```

Splitting by string should be faster than regex. Also switched to use three dots like github

Reply:
Switched regex to string.